frontend/src/scenes/surveys/SurveyDragHandle.tsx

```diff
@@ -4,6 +4,7 @@ import { SortableDragIcon } from 'lib/lemon-ui/icons'
 interface SurveyDragHandleProps {
     listeners: DraggableSyntheticListeners | undefined
     isDraftSurvey: boolean
+    allQuestionsHaveIds: boolean
```

Comment:
do we actually need this? we ran the script adding IDs, so I don't see how this is ever returning false